sdk/core/azure-core-tracing-opentelemetry-samples/pom.xml

```diff
@@ -31,6 +31,10 @@
     <tag>HEAD</tag>
   </scm>
 
+  <properties>
+    <checkstyle.suppressionsLocation>checkstyle-suppressions.xml</checkstyle.suppressionsLocation>
+  </properties>
+
```

Comment:
Are we not supposed to run SpotBugs cheks on sample projects?

Reply:
Or is the SpotBugs configuration coming from the parent SDK supposed to have it run and we don't want to allow it to be disabled like in other POMs?

Reply:
Since this is just samples I left Spotbugs disabled for now